### ISSUE TITLE
lfw evaluation

### ISSUE THREAD

**Issue by Baicheng42** [suzhigangssz/AVIH#4]
Hi, Thanks for your great work!

I met some questions about evaluations of LFW dataset. I used your Arcface model here and the code of Dataloader and Test protocol from [https://github.com/QuasarLight/Pytorch_Face_Recognition-](url), but I failed to get a high accuracy. The acc is only 67.33 even on the unprotected data i.e. original face images.

Could you please provide me a link about how to prepare the dataloader and test code. Thank you very much!

Best regard.

**Comment by suzhigangssz** [comment]
The input range of the ArcFace we referenced is [0,255], so don't forget to adjust the samples before feeding them into the model.  Also we fixed a small bug in line 30 of the avih.py file. Note that the testing method in our paper is different from the common way of testing lfw (see Section 4.1 Evaluation metrics in the paper for more details), but this does not affect the effectiveness of the method, and you can choose this method for evaluation.

**Comment by suzhigangssz** [comment]
> Hi, Thanks for your great work!
> 
> I met some questions about evaluations of LFW dataset. I used your Arcface model here and the code of Dataloader and Test protocol from [https://github.com/QuasarLight/Pytorch_Face_Recognition-](url), but I failed to get a high accuracy. The acc is only 67.33 even on the unprotected data i.e. original face images.
> 
> Could you please provide me a link about how to prepare the dataloader and test code. Thank you very much!
> 
> Best regard.

The input range of the ArcFace we referenced is [0,255], so don't forget to adjust the samples before feeding them into the model. Also we fixed a small bug in line 30 of the avih.py file. Note that the testing method in our paper is different from the common way of testing lfw (see Section 4.1 Evaluation metrics in the paper for more details), but this does not affect the effectiveness of the method, and you can choose this method for evaluation.

**Comment by Baicheng42** [comment]
Thank you again! I ignored that since I downloaded the previous version of this code, and now I have fixed this.
